src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -4953,24 +4953,68 @@ static void Main()
         }
 
         [Fact]
-        public void TupleLiteralElement004()
+        public void TupleLiteralElement004_WithoutValueTuple()
         {
             var source =
 @"
+class C
+{
+    static void Main()
+    {
+        (short X, string Y) = (Alice: 1, Bob: null);
+    }
+}
+";
 
-using System;
+            var compilation = CreateStandardCompilation(source);
+            compilation.VerifyDiagnostics(
+                // (6,31): error CS8179: Predefined type 'System.ValueTuple`2' is not defined or imported, or is declared in multiple referenced assemblies
+                //         (short X, string Y) = (Alice: 1, Bob: null);
+                Diagnostic(ErrorCode.ERR_PredefinedValueTupleTypeNotFound, "(Alice: 1, Bob: null)").WithArguments("System.ValueTuple`2").WithLocation(6, 31)
+                );
+            var tree = compilation.SyntaxTrees[0];
+            var decl = (ArgumentSyntax)tree.GetCompilationUnitRoot().DescendantNodes().Last(n => n.IsKind(SyntaxKind.Argument));
+            var model = compilation.GetSemanticModel(tree);
+            Assert.Null(model.GetDeclaredSymbol(decl));
```

Comment:
:memo: There is a slight regression of the semantic model here. But (1) I don't think it is very significant, and (2) I think it is worthwhile because the root cause is a change to avoid many duplicate diagnostics.